doc/conf.py

```diff
@@ -97,18 +97,20 @@
             'title': 'Component Gallery',
             'sections': [
                 'panes',
```

Comment:
I propose this order because I believe Panes and widgets are most used. The Panes on top because they can quickly give you a visual impression of what Panel supports if you are a new user. Layouts at bottom because I believe they are lesser used?